scripts/android_e2e.sh

```diff
@@ -1,20 +1,19 @@
 #!/bin/bash
```

Comment:
Would be nice if we could convert these scripts to JS so we're tied to having a Mac.

Reply:
Good point, but let's do it as a follow up PR